doc/syscalls/20006_gpio.md

```diff
@@ -1,5 +1,5 @@
 ---
-driver number: 0x00004
+driver number: 0x20006
```

Comment:
This isn't your fault, and apparently it's true for all the drivers, but why does this number appear in decimal in the rendered document view?

Reply:
I'm not sure where this is rendered (and what to actually), but it does look a lot like a YAML front matter, so probably this is just interpreted as a simple variable containing an integer - parsed from hex.